sparta/test/PEvents/PEventHelper_test.cpp

```diff
@@ -97,6 +97,7 @@ int main() {
     trigger.go();
     A a(1000, 78, 52, 0.01, "test0");
     pair_pevent.collect(a);
+    EXPECT_THROW(pair_event.collect(a, "unexpected positional pair arg"));
```

Comment:
```suggestion
    EXPECT_THROW(pair_pevent.collect(a, "unexpected positional pair arg"));
```
Helps if I actually use the correct variable for the test. 🤦‍♂️